src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java

```diff
@@ -57,7 +92,15 @@ public void startProcessInstance() {
         variables.put(BENCHMARK_START_DATE_MILLIS, Instant.now().toEpochMilli());
         variables.put(BENCHMARK_STARTER_ID, config.getStarterId());
 
-        // Auto-complete logic from https://github.com/camunda-community-hub/spring-zeebe/blob/ec41c5af1f64e512c8e7a8deea2aeacb35e61a16/client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/jobhandling/JobHandlerInvokingSpringBeans.java#L24
```

Comment:
Why do you remove this comment? Please leave it on top of the code that it was on.